### ISSUE TITLE
It's unclear which documentation *is* supported

### ISSUE THREAD

**Issue by jpluscplusm** [cloudfoundry-incubator/kubo-release#256]
This text from this repo's readme leaves me with some questions:

```
We are no longer supporting the following documentation for deploying BOSH and CFCR

- https://docs-cfcr.cfapps.io
```

Does that mean the linked site should be considered out of date, potentially bit-rotting, and shouldn't be used?

Does it mean that https://github.com/cloudfoundry/docs-cfcr is deprecated?

Which docs *are* supported - just the contents of this repo's readme?

**Comment by seanos11** [comment]
Hi @jpluscplusm 
Thanks for the feedback

We posted the deprecation notice during a docs migratoin period. We recently took down the content in https://docs-cfcr.cfapps.io that is no longer supported, so leaving the deprecation notice meaningless, I will clean that up

To clarify, we are building on the README.md and pages in /docs folder of kubo-release as our primary customer onboarding/product-reference content.

 https://docs-cfcr.cfapps.io remains as a product introduction page and where the release notes continued to be published.

Sean
